app/api_models/users.py

```diff
@@ -0,0 +1,79 @@
+import datetime
+from werkzeug.security import generate_password_hash, check_password_hash
+from app import db
+from app.api_models.shoppinglist import ShoppingList
+import jwt
+from flask import current_app
+from app.api_models.logout import BlacklistToken
+
```

Comment:
There is a better python way of making imports 
The link below can help
https://www.python.org/dev/peps/pep-0008/#imports